contracts/single-party-pol-covenant/src/contract.rs

```diff
@@ -194,8 +162,8 @@ pub fn instantiate(
     let preset_holder_fields = PresetHolderFields {
         code_id: msg.contract_codes.holder_code,
         label: format!("{}-holder", msg.label),
-        withdrawer: None,
-        withdraw_to: None,
+        withdrawer: Some(info.sender.to_string()),
+        withdraw_to: Some(info.sender.to_string()),
```

Comment:
The withdrawer / withdarw_to can be not the sender, in this case we rather get it as field, and also set it to be not optional.
The reason I kept it optional is because the parties can start the covenant and decide on what the addresses are at later point of time.